lib/membership_row.rb

```diff
@@ -0,0 +1,37 @@
+require 'scraped'
+
+class MembershipRow < Scraped::HTML
+  TERM_IDS_MAP = {
+    'Legislatura' => '0',
+    'I' => '1',
+    'II' => '2',
+    'III' => '3',
+    'IV' => '4',
+    'V' => '5',
+    'VI' => '6',
+    'VII' => '7',
+    'VIII' => '8',
+    'IX' => '9',
+    'X' => '10',
+    'XI' => '11',
+    'XII' => '12'
+  }.freeze
```

Comment:
It would be good to be more consistent with ParsedDateString here, rather than one case using a Hash lookup, and the other find_index-ing into an Array. The Hash is almost certainly faster, but that's almost certainly also premature optimisation, so I'd tend to lean towards the one that's shorter.